client/web/web.go

```diff
@@ -756,6 +732,53 @@ func (s *Server) tailscaleUp(ctx context.Context, st *ipnstate.Status, postData
 	}
 }
 
+type tailscaleUpOptions struct {
+	Reauthenticate bool
+	ForceLogout    bool
```

Comment:
Do we actually need logout from this endpoint? I think we're using the local api proxy for logout? Oh I guess that's not available from the login view

Reply:
I had removed it, but the added it back in case we might need it in the future.  I forgot that (I'm pretty sure) we had decided that we'd only support logout from the full management client.  So we definitely don't need this here.  Removed.